.github/workflows/build.yml

```diff
@@ -126,6 +126,7 @@ jobs:
 
   ios_integration_test:
     runs-on: macos-latest
+    timeout-minutes: 600
```

Comment:
10 hours?! Whoa! I think default 360 will be enough :)

Reply:
We have "TimeoutException after 0:12:00.000000: Test timed out after 12 minutes." exception, that points us to completely separated problem:
https://github.com/flutter/flutter/issues/103064

Reply:
-